HaptikLib.podspec

```diff
@@ -42,12 +42,11 @@ Pod::Spec.new do |s|
     base.vendored_frameworks = "Frameworks/Base/HaptikBase.framework"
   end
 
-  # Haptik Payment Module Subspec
+  # Haptik Location Module Subspec
 
-  s.subspec 'Payments' do |payments|
-    payments.ios.deployment_target = '9.0'
-    payments.dependency    'PayUIndia-Custom-Browser', '5.8.1'
-    payments.vendored_frameworks = "Frameworks/Payments/HaptikPayments.framework"
+  s.subspec 'Location' do |location|
+    location.ios.deployment_target = '10.0'
+    location.vendored_frameworks = "Frameworks/Location/HaptikLocation.framework"
```

Comment:
Don't we need to weakly link the frameworks for the `CoreLocation` here?

Reply:
`CoreLocation`/`MapKit`  are auto-linked so we don't need to specify them.
similar to `CoreData`.
we can add these as a preventive measure tho

Reply:
Cool